README.md

```diff
@@ -42,24 +42,20 @@ Add .boto to your environment
 Open .boto and Add (`/etc/boto.cfg` for main)
 
     [Credentials]
-    aws_access_key_id = {keyid}
-    aws_secret_access_key = {accesskey}
+    aws_access_key_id = ${keyid}
+    aws_secret_access_key = ${accesskey}
 
 [Nagios Check for Dynamodb Dumps](https://dashboard.bgdi.ch/cgi-bin/nagios3/extinfo.cgi?type=2&host=ip-10-220-4-46.eu-west-1.compute.internal&service=DynamoDB+backup)
 
-Bootstrap your build environment:
-
-    python bootstrap.py --version 1.5.2 --distribute --download-base http://pypi.camptocamp.net/distribute-0.6.22_fix-issue-227/ --setup-source http://pypi.camptocamp.net/distribute-0.6.22_fix-issue-227/distribute_setup.py
-
 Create a developer specific build configuration:
 
-    cp buildout_ltgal.cfg buildout_<username>.cfg
+    cp rc_example rc_user_<username>.cfg
```

Comment:
Because we don't want to commit the personal configs in the main project, for clarity sake and because of make user target.